packages/react-recommendations/README.md

```diff
@@ -176,17 +252,93 @@ The product component to display.
 > `Translations`
```

Comment:
```suggestion
> `RecommendationTranslations`
```

Reply:
Done in https://github.com/algolia/ui-components/pull/18/commits/a1991cfb5cb8bc7bd4502425134439493283c578